<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(255, 255, 255, 100), stop:1 rgb(188, 188, 188));</string>
     </property>
     <property name="text">
      <string>검색(Search)</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>151</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
     </property>
     <property name="title">
      <string>검색할 정보</string>
     </property>
     <widget class="QRadioButton" name="byCategory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>108</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>업종별 검색</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="byArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>108</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>지역별 검색</string>
      </property>
     </widget>
     <widget class="QLabel" name="cityLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>시군구</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cityBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="dongLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>법정동</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dongBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCategoryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>업종대분류</string>
      </property>
     </widget>
     <widget class="QComboBox" name="LCategoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="MCategoryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>업종중분류</string>
      </property>
     </widget>
     <widget class="QComboBox" name="MCategoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SCategoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="SCategoryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>업종소분류</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>640</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(255, 255, 255, 100), stop:1 rgb(188, 188, 188));</string>
     </property>
     <property name="text">
      <string>뒤로가기(Back)</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>90</y>
       <width>1000</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>1111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Table</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
